src/api/common/get-headers.js

```diff
@@ -11,7 +11,7 @@ export default function (method, okapi, url) {
   const headers = {
     'Content-Type': 'application/json',
     'X-Okapi-Tenant': okapi.tenant,
-    'X-Okapi-Token': okapi.token
+    ...(okapi.token && { 'X-Okapi-Token': okapi.token }),
```

Comment:
From the ticket description it looks like we need to completely remove `X-Okapi-Token`

Reply:
I think we can leave it according to this comment [comment](https://issues.folio.org/browse/UILDP-77?focusedCommentId=160520&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-160520).